zfit/z/numpy.py

```diff
@@ -10,14 +10,140 @@
+            -66155.39578477248,
+            57778.05827983565j,
+            +11682.770904216826,
+            -9442.402767960672j,
+            -1052.8438624933142,
+            814.0996198624186j,
+            +45.94499030751872,
+            -34.59751573708725j,
+            -0.7616559377907136,
+            0.5641895835476449j,
+        ],
+        dtype=complex128,
+    )
+
+    bb = array(
+        [
+            +7918.06640624997,
+            -126689.0625,
+            +295607.8125,
+            -236486.25,
+            +84459.375,
+            -15015.0,
+            +1365.0,
+            -60.0,
+            +1.0,
+        ],
+        dtype=complex128,
+    )
+
+    sqrt_piinv = 1.0 / _np.sqrt(_np.pi)
+
+    zz = z * z
+    w = 1j * z * (zz * sqrt_piinv - 1.410474) / (0.75 + zz * (zz - 3.0))
+
+    real_part = real(z)
+    imag_part = imag(z)
+    mask = logical_and(abs(real_part) + imag_part < s, imag_part < s)
+
+    Z = z[mask] + 1.35j
+    ZZ = Z * Z
+
+    numer = (
+        (
+            (
+                (
+                    (
+                        (
+                            (
+                                (
+                                    (
+                                        (
+                                            (
+                                                (
+                                                    ((AA[15] * Z + AA[14]) * Z + AA[13])
+                                                    * Z
+                                                    + AA[12]
+                                                )
+                                                * Z
+                                                + AA[11]
+                                            )
+                                            * Z
+                                            + AA[10]
+                                        )
+                                        * Z
+                                        + AA[9]
+                                    )
+                                    * Z
+                                    + AA[8]
+                                )
+                                * Z
+                                + AA[7]
+                            )
+                            * Z
+                            + AA[6]
+                        )
+                        * Z
+                        + AA[5]
+                    )
+                    * Z
+                    + AA[4]
+                )
+                * Z
+                + AA[3]
+            )
+            * Z
+            + AA[2]
+        )
+        * Z
+        + AA[1]
+    ) * Z + AA[0]
+
+    denom = (
+        (
+            (((((ZZ + bb[7]) * ZZ + bb[6]) * ZZ + bb[5]) * ZZ + bb[4]) * ZZ + bb[3])
+            * ZZ
+            + bb[2]
+        )
+        * ZZ
+        + bb[1]
+    ) * ZZ + bb[0]
+
+    w = _tf.tensor_scatter_nd_update(w, _tf.where(mask), numer / denom)
```

Comment:
```suggestion
    w = _tf.tensor_scatter_nd_update(w, where(mask), numer / denom)
```
where is in znp namespace

Reply:
I actually reverted this commit. Tensorflow `where` is different from the numpy `where`. We want the tensorflow one there.

Reply:
Ah yes? It's just the shape though I think?